src/api/include/migraphx/migraphx.hpp

```diff
@@ -1255,7 +1299,9 @@ struct experimental_custom_op_base
     virtual std::string name() const                                            = 0;
     virtual argument compute(context ctx, shape output, arguments inputs) const = 0;
     virtual shape compute_shape(shapes inputs) const                            = 0;
-    virtual ~experimental_custom_op_base()                                      = default;
+    virtual ptrdiffs output_alias(shapes) const { return {-1}; }
```

Comment:
The default should be empty. Empty means it doesn't alias anything. We should also make this be `std::vector<std::size_t>` instead of using signed integers.